### PR TITLE
finished code challenge 7

### PR DESCRIPTION
**Pull request by PotatoGod123**
- [ x] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [x ] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [x ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [ x] Link to code
     - [ x] Picture of whiteboard

### CODE REVIEW

File: challenges/test/lib/src/test/java/code/linkedlisttest/LinkedListTest.java [hunk 1]
```diff
@@ -103,4 +103,20 @@
 
   }
 
+  @Test public void testKFromTheEnd() throws Exception {
```
Comment (Mattpet26) [inline challenges/test/lib/src/test/java/code/linkedlisttest/LinkedListTest.java:106]: Don't forget to test all of your methods!
